functions/validator/main.py

```diff
@@ -33,6 +33,7 @@ def handle(event, context):
     if payload_status is True:
         logger.info("Payload is valid sending to kinesis.")
         # To-Do Write to kinesis
+        streams.publish_to_cis(data=encryption.decrypt(**event), partition_key='mozillians.org')
```

Comment:
Yep.  This would likely be supplied by the publisher as part of the initial event payload.  But requires more conversation to know what that invoke call will look like.